domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -87,4 +88,10 @@ object InteractionObjectTestBuilder {
   fun createInt(value: Int): InteractionObject {
     return InteractionObject.newBuilder().setReal(value.toDouble()).build()
   }
+
+  public fun createRatio(value: List<Int>): InteractionObject {
```

Comment:
As the InteractionObjectTestBuilder is an object, we don't need to specify public here.

Reply:
done